modules/sagemaker/outputs.tf

```diff
@@ -1,11 +1,4 @@
-output "model_name" {
-  value = var.model_name
+output "notebook_instance" {
+  description = "Sagemaker notebook instance Terraform object"
+  value       = aws_sagemaker_notebook_instance.notebook_instance
 }
-
-output "sagemaker_endpoint_name" {
```

Comment:
It seems useless to pass unchanged variables as outputs - the user already has these, as they have passed them in

Reply:
I removed some of the outputs because we were just passing variables the user had supplied back to them again - but since refactoring the naming conventions, I have added this output back

Reply:
Done